ECommerceDemoCommon/ECommerceDemoCommon.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What is the purpose of this project?

Reply:
ECommerceDemoInfrastructure (previously named as ECommerceDemoCommon) contains
"Infrastructure" classes like interfaces, entities, data provider classes,
which can be re-used by several  WebAPI endpoint projs.